src/annotator/adder.js

```diff
@@ -180,9 +181,11 @@ export class Adder {
     // Set the initial arrow direction based on whether the selection was made
     // forwards/upwards or downwards/backwards.
     /** @type {ArrowDirection} */ let arrowDirection;
-    if (isRTLselection) {
+    if (isRTLselection && !isTouchDevice()) {
       arrowDirection = ARROW_POINTING_DOWN;
     } else {
+      // Render the adder below the selection for touch devices due to competing
+      // space with the copy/paste bar.
```

Comment:
I think this sentence could be clarified to explain that "the copy/paste bar" is a native UI which typically appears above the selection on touch devices.